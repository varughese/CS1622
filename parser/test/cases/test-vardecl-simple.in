int var1;
void main(void) {}
