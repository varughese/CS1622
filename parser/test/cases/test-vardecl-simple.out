[program
  [var-declaration [int] [var1]]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
    ]
  ]
]
